src/Atc.Rest.Client/Atc.Rest.Client.csproj

```diff
@@ -30,8 +30,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.3.0" />
-    <PackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.1.0" />
+    <PackageReference Include="System.Text.Json" Version="5.0.0" />
```

Comment:
Question - the targetFramework is netstandard2.0, will this work with System.Text.Json ?